lib/ngx/ssl.md

```diff
@@ -369,6 +369,11 @@ Sets the SSL certificate chain opaque pointer returned by the
 
 Returns `true` on success, or a `nil` value and a string describing the error otherwise.
 
+Note that the `set_cert` function will perform slightly better, in terms of CPU cycles wasted, than then
+[set_der_cert](#set_der_cert) variant, since the first function uses opaque cdata pointers
+which do not require any additional conversion needed to be performed by the NGINX core during the SSL session
```

Comment:
Actually the additional conversion is done by OpenSSL instead of the NGINX core. Better be more accurate here.